pisek/utils/terminal.py

```diff
@@ -25,6 +25,12 @@
 except OSError:
     terminal_width, terminal_height = 100, 24
 
+TARGET_LINE_WIDTH = min(terminal_width, 100)
+
+
+def right_aligned_text(left: str, right: str, offset: int = 0):
```

Comment:
Nit: Is this a good name? It doesn't *just* right align text, it also left aligns some.

Reply:
I'm not sure `split_aligned_text` is that great, either, as it (at least to me) sounds like it splits something.
I can't really think of a good name for the function. Maybe `right_aligned_text` is the best option after all?